i18n/en.json

```diff
@@ -6,11 +6,11 @@
 			"Agent Isai"
 		]
 	},
-	"action-createwiki": "create wikis",
-	"action-createwiki-deleterequest": "delete wiki requests",
-	"action-createwiki-suppressionlog": "view the wiki request suppression log",
-	"action-createwiki-suppressrequest": "suppress wiki requests",
-	"action-requestwiki": "request wikis",
+	"action-createwiki": "Create wikis",
```

Comment:
These messages should remain as is. They are used in special pages when you don't have the required user right to use them.

With these changes, there would be a random string in uppercase for no apparent reason. The message would look like "You don't have permission to Create wikis".